jte-models/src/main/java/gg/jte/models/generator/ModelExtension.java

```diff
@@ -21,7 +21,7 @@ public class ModelExtension implements JteExtension {
 
     @Override
     public String name() {
-        return "Generate type-safe model facade for templates";
+        return "Generate type-safe model facade for templates in Java";
```

Comment:
I'm also unsure about the impact of changing this.

Reply:
The name isn't actually used anywhere at the moment.